optuna/samplers/_tpe/sampler.py

```diff
@@ -751,8 +752,11 @@ def _split_complete_trials_multi_objective(
         need_tiebreak = nondomination_ranks == last_rank_before_tiebreak + 1
         rank_i_lvals = lvals[need_tiebreak]
         subset_size = n_below - indices_below.size
-        selected_indices = _solve_hssp(
-            rank_i_lvals, indices[need_tiebreak], subset_size, _get_reference_point(rank_i_lvals)
+        selected_indices = solve_hssp_with_cache(
+            tuple(map(tuple, rank_i_lvals)),
```

Comment:
This is quicker compared to using `map(tuple, ...)`.
```suggestion
            tuple(rank_i_lvals.ravel()),
```